app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="recipes">
+<%= link_to "Add recipe", new_recipe_path, class: "btn btn-secondary"%>
+<% @recipes.each do |recipe|%>
+<div class="recipe">
+  <div class="recipe-name">
+    <div>
+    <%= link_to "#{recipe.name}", recipe_path(recipe), id: "recipe_#{recipe.id}", class: "text-reset text-decoration-none fs-3 text-capitalize" %>
+    </div>
+    <div class="remove-recipe">
+      <%= link_to 'Remove', recipe_path(recipe), method: :delete, class: "badge bg-danger text-decoration-none" %>
+    </div>
```

Comment:
- [ ] Deleting this recipe gives a routing error. Kindly fix it and ensure that only the user that created the recipe can delete it.